sdk/storage/platform-matrix-all-versions.json

```diff
@@ -0,0 +1,27 @@
+{
+  "matrix": {
+    "Agent": {
+      "ubuntu-18.04": {
+          "OSVmImage": "MMSUbuntu18.04",
+          "Pool": "azsdk-pool-mms-ubuntu-1804-storage"
+      },
+      "windows-2019": {
+          "OSVmImage": "MMS2019",
+          "Pool": "azsdk-pool-mms-win-2019-storage"
+      }
+    },
```

Comment:
We will add macOS back in to live tests at the same time we add it in for all NET live tests (due to ongoing capacity issues with devops hosted mac pools).